aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -800,9 +887,7 @@ module aptos_framework::account {
         let account = borrow_global_mut<Account>(account_addr);
         event::emit_event<CoinRegisterEvent>(
             &mut account.coin_register_events,
-            CoinRegisterEvent {
-                type_info: type_info::type_of<CoinType>(),
-            },
+            CoinRegisterEvent { type_info: type_info::type_of<CoinType>(), },
```

Comment:
Superflous comma in the struct definition, but that's no big deal.

Reply:
We have taken note of the issue while addressing https://github.com/movebit/movefmt/issues/11, and we will continue to optimize this issue in the future.